Sources/TUSKit/Tasks/IdentifiableTask.swift

```diff
@@ -6,7 +6,3 @@
 //
 
 import Foundation
-
-protocol IdentifiableTask: ScheduledTask {
-    var id: UUID { get }
```

Comment:
Removing this protocol allows the tasks to no longer have an id value.
https://docs.swift.org/swift-book/LanguageGuide/Protocols.html

It looks like this may have been due to adding taskType. You could keep the IdentifiableTask protocol by adding taskType to the definition.

`var taskType: string { get }`

Reply:
The id has been moved into the ScheduledTask itself. 
If you try to create a class that implements the ScheduledTask protocol without defining an id it will complain still.
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/92157216/178275772-52cb0bef-4f22-4b83-a4ad-621a20b300a0.png">

Reply:
ah, good to know